lib/web_ui/lib/src/engine/pointer_binding/event_position_helper.dart

```diff
@@ -25,6 +29,12 @@ ui.Offset computeEventOffsetToTarget(DomMouseEvent event, DomElement actualTarge
   }
 
   final bool isTargetOutsideOfShadowDOM = event.target != actualTarget;
+  final bool isInput = event.target.toString() == 'input';
```

Comment:
This won't work for `<textarea>`. Instead let's check if `event.target` is a descendant of the text editing host.

```suggestion
  final bool isInput =
      flutterViewEmbedder.textEditingHostNode.contains(event.target);
```